### ISSUE TITLE
PHP PEAR in "vendor/pear-pear.php.net" Is Outdated

### ISSUE THREAD

**Issue by zeroflags** [roundcube/roundcubemail#5171]
When trying to use the "Password" plugin with LDAP server, the following error occurs:
`PHP Fatal error:  Access level to Net_LDAP2_RootDSE::__construct() must be public (as in class PEAR) in /usr/local/www/roundcube/vendor/pear-pear.php.net/Net_LDAP2/Net/LDAP2/RootDSE.php on line 238`
If I use the system (FreeBSD 10.2) provided PHP PEAR library, it works normally. But another issue occurs:
`PHP Fatal error:  Access level to Net_LDAP2_Schema::__construct() must be public (as in class PEAR) in /usr/local/www/roundcube/vendor/pear-pear.php.net/Net_LDAP2/Net/LDAP2/Schema.php on line 621`.

Obviously, the issue is [not new](http://www.roundcubeforum.net/index.php?topic=23182.0) and the only thing that would be necessary to do is to update the "Complete" version accordingly.

I also suggest to respect the directory structure of the PHP PEAR library as it would be easier to track the changes. Just my two cents... Thanks.

**Comment by alecpl** [comment]
Indeed, we should require Net_LDAP2 ~2.2.0. However, there's currently a bug in PHP7 that makes this version not possible to install using composer.
```
Failed to extract PEAR package /vendor/pear-pear.php.net/Net_LDAP2/Net_LDAP2-2.2.0.tgz to /vendor/pear-pear.php.net/Net_LDAP2. Reason: phar error: tar-based phar "Net_LDAP2-2.2.0/tests/ldif_data/malformed_syntax.ldif" cannot be registered
```
Looks like it might be fixed in PHP 7.0.5 (https://bugs.php.net/bug.php?id=71317).

**Comment by alecpl** [comment]
I pushed the change anyway.
